src/main/webapp/index.html

```diff
@@ -26,11 +26,15 @@
           <p>Enter a YouTube video link:</p>
           <input id="link-input" name="link-input" required pattern="https?://www.youtube.com/watch\?v=.+"/>
           <small id="url-help" class="form-text text-muted">Format: https://youtube.com/watch?v=...</small>
+          <p>Enter the maximum number of comments to be analyzed: </p>
+          <input type="range" id="numComments-input" name="numComments-input" min="100" max="1000" value = "100" step="10"/>
```

Comment:
Just a minor thing: can we make this a little bit bigger in the future:)

Reply:
Make what a little bigger?